### PR TITLE
37 list rfi

### PR DESCRIPTION
**Pull request by marcellmueller**
Some things to note: 

- This doesn't have an onChange handler for saving as the mutation is being created in edit rfi ticket
- There is no way to do real world testing of listing the applicants files that were filled after being requested as that hasn't been developed yet. Taking that into consideration it was developed and tested with mock data.

### CODE REVIEW

**Review by AntBush — APPROVED** [review]
Looks good! Works well locally, just a couple of comments/nits

File: app/components/Analyst/RFI/DefaultWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  min-width: 140px;
+`;
+
+const StyledColumnRight = styled.div`
+  padding-top: 0;
+`;
+
+export const HiddenWidget = () => {
+  // eslint-disable-next-line react/jsx-no-useless-fragment
+  return <></>;
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply app/components/Analyst/RFI/DefaultWidget.tsx]: Nice, that worked.

File: app/backend/lib/s3download.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ResponseContentDisposition: `attachment; filename="${fileName}"`,
+  });
+
+  signedUrl
+    .then((url) => {
+      res.json(url);
+    })
+    .catch(() => {
+      res.status(500).end();
+    });
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply app/backend/lib/s3download.ts:36]: I fixed this but had to return `signedUrl`. It expects a return value in the block scope of the function so returns inside if statements or then/catch blocks don't work. Thanks :+1:

File: app/tests/pages/analyst/application/[applicationId]/rfi/index.test.tsx [hunk 3]
```diff
@@ -75,6 +136,70 @@ describe('The index page', () => {
     ).toBeVisible();
   });
 
+  it('lists the RFIs', async () => {
+    pageTestingHelper.loadQuery();
+    pageTestingHelper.renderPage();
+
+    expect(
+      screen.getByRole('heading', { name: 'CCBC-010001-1' })
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply app/tests/pages/analyst/application/[applicationId]/rfi/index.test.tsx:144]: I can't seem to replicate this, it's working on my machine :thinking:

**Review by VB-at-Bis — APPROVED** [review]
Looks safe

**Pull request merged**